xcube/core/store/descriptor.py

```diff
@@ -65,6 +65,16 @@ class DataDescriptor:
     """
     A generic descriptor for any data.
     Also serves as a base class for more specific data descriptors.
+
```

Comment:
Some of my comments on the DatasetDescriptor docstrings also apply to the corresponding parameters here, but it wasn't worth duplicating them.